README.md

```diff
@@ -1,501 +1,69 @@
-# Workflow Description Language (WDL)
+Workflow Description Language (WDL)
+========================================
 
-WDL is a workflow language meant to be read and written by humans. Broader documentation is provided
-by the [WDL website](https://software.broadinstitute.org/wdl/). Any questions or issues can be discussed at
-our [support forum](http://gatkforums.broadinstitute.org/wdl).
+The **Workflow Description Language (WDL)** is a way to specify data processing workflows with a human-readable and writeable syntax. WDL makes it straightforward to define complex analysis tasks, chain them together in workflows, and parallelize their execution. The language makes common patterns simple to express, while also admitting uncommon or complicated behavior; and strives to achieve portability not only across execution platforms, but also different types of users. Whether one is an analyst, a programmer, an operator of a production system, or any other sort of user, WDL should be accessible and understandable.
 
-* [Official Language Specification](https://github.com/openwdl/wdl/blob/master/versions/1.0/SPEC.md) 
-* [Developmental Language Specification](https://github.com/openwdl/wdl/blob/master/versions/development/SPEC.md) 
+# Language Specifications:
 
+The current version of the WDL language is **1.0**. The [1.0 specification](https://github.com/openwdl/wdl/blob/master/versions/1.0/SPEC.md) contains all relevant information for users, developers, and engine developers. Upcoming features which have previously been accepted can be viewed as part of the [development spec](https://github.com/openwdl/wdl/blob/master/versions/development/SPEC.md). Additional language versions are listed below.
 
-Library and engine support is provided by
 
-* [Java parser](parsers/java) which provides only a parser to convert a WDL string into an AST
-* [wdl4s](http://github.com/broadinstitute/wdl4s) provides Scala bindings for WDL and uses the above Java parser
-* [PyWDL](https://github.com/broadinstitute/pywdl) provides Python bindings for WDL
-* [Cromwell](http://github.com/broadinstitute/cromwell) is an engine for running WDL workflows.  This uses [wdl4s](http://github.com/broadinstitute/wdl4s)
+### All Version
+* [1.0](https://github.com/openwdl/wdl/blob/master/versions/1.0/SPEC.md)
+* [development](https://github.com/openwdl/wdl/blob/master/versions/development/SPEC.md)
+* [Draft-3](https://github.com/openwdl/wdl/blob/master/versions/development/SPEC.md)
```

Comment:
I'm not sure it's worth having these draft versions here. If you think it's good to keep them I'd draw attention to the fact that they're old/obsolete